src/utils/reward-utils.ts

```diff
@@ -0,0 +1,116 @@
+import { BigNumber, BigNumberish, ethers } from "ethers";
+import request, { gql } from "graphql-request";
+import config from "../config";
+import { BASIS_POINTS_DIVISOR, PRICE_DECIMALS } from "../config/hlp";
+import { GovernanceLock__factory, RewardPool__factory } from "../contracts";
+import { Network } from "../types/network";
+
+export const calculateUserAPR = (
+  userBoost: number,
+  averagePoolBoost: number,
+  poolAPR: number
+): number => {
+  return (poolAPR * userBoost) / averagePoolBoost;
+};
+
+type FeeStats = {
+  marginAndLiquidation: string;
+  swap: string;
+  mint: string;
+  burn: string;
+};
+type GraphResponse = {
+  feeStats: FeeStats[];
+  hlpStats: {
+    aumInUsdg: string;
+  }[];
+};
+
+export const fetchFeeAPR = async (
```

Comment:
camelCase for function names please
```suggestion
export const fetchFeeApr = async (
```